src/components/UpdateContract.tsx

```diff
@@ -14,10 +14,10 @@ interface UpdateContractProps {
 
 const UseContract = ({ Tezos }: UpdateContractProps) => {
   const base62 = require("base62-random");
```

Comment:
we should move this to the top of the script.
Also, this can be replaced with `import base62 from "base62-random";` I think.
Currently this module will be loaded from cache on every re-render of the component.